apps/sms/test/marionette/lib/views/new-message/view.js

```diff
@@ -167,6 +167,12 @@ NewMessageView.prototype = {
     return this.composerAccessors.messageInput.scriptWith(function(el) {
       return document.activeElement === el;
     });
+  },
+
+  isRecipientsInputFocused: function() {
+    return this.accessors.recipientsInput.scriptWith(function(el) {
+      return document.activeElement === el;
+    });
```

Comment:
wondering if we should have a more generic "isElementFocused" that we could reuse. I don't know where we can have such a toolset though... @azasypkin what do you think ?